modules/sdk-coin-ada/src/lib/transactionBuilder.ts

```diff
@@ -169,9 +168,10 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     );
 
     // calculate the fee based off our dummy transaction
-    if (!this._fee) {
+    if (!this._transaction.getFee || this._transaction.feeSetFromRawTransaction) {
       const fee = CardanoWasm.min_fee(txDraft, linearFee).checked_add(BigNum.from_str('440'));
-      this._fee = fee;
+      this._transaction.fee(fee);
```

Comment:
if we're replacing all of the TransactionBuilder.fee instances with Transaction.fee, maybe we can just get rid of that property in TransactionBuilder